### DISCUSSION TITLE
Please use a gs:// URL for your Firebase Storage bucket.

### DISCUSSION THREAD

**Discussion by AhmadAl-Ghalban** [invertase/react-native-firebase#6152] [Q&A]
Hi everybody.

When I upload a file using Android the storage works fine.
The basic functionality example has taken from the official documentation page.
THis is Code => 

import storage from '@react-native-firebase/storage';

export const saveMediaToStorage = (media, path) => new Promise((resolve, reject) => {
    const fileRef = storage().ref().child(path)
    console.log('media', media);
    fetch(media)
        .then(response => response.blob())
        .then(blob =>
            // console.log('blob', blob);
            fileRef.put(blob)
        ).then((task) => {
            console.log('Start Task ');
            task = storage().ref().getDownloadURL()
            console.log('end  Task ');
            console.log('task', task)
}
        ).then(downloadUrl => {
            console.log('downloadUrl ', downloadUrl)
            resolve(downloadUrl)
}
        ).catch((error) => {
            console.log('error SaveMedia is => ', error);
            reject()
        })

})

the problem is =>

![ee](https://user-images.githubusercontent.com/70822571/159840980-2d43a4e9-68f5-4b0b-92cd-a96eb501110c.PNG)

**Comment by mikehardy** [comment]
Sounds like your path does not start with `gs://`
